<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy Uploader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>my_app.icns</normaloff>my_app.icns</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QComboBox" name="cboCommPort"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnUploadFlash">
       <property name="text">
        <string>Upload...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOutput">
       <property name="text">
        <string>1) Select Comm Port 2) Click Upload... 3) Select .hex file</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShowConsole">
       <property name="text">
        <string>Show Console</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
